### DISCUSSION TITLE
Execute method constantly keep executing

### DISCUSSION THREAD

**Discussion by jsonp80** [quartznet/quartznet#948] [General]
I follow example (https://andrewlock.net/creating-a-quartz-net-hosted-service-with-asp-net-core/)  and change schedule to 5 minutes  but when i run in visual studio it keep executing this Execute method every second. my cron job should run every 5 minutes.  Is this normal?

 public class HelloWorldJob : IJob
{
    // Inject the DI provider
    private readonly IServiceProvider _provider;
    public HelloWorldJob( IServiceProvider provider)
    {
        _provider = provider;
    }

    public Task Execute(IJobExecutionContext context)
    {
        // Create a new scope
        using(var scope = _provider.CreateScope())
        {
            // Resolve the Scoped service
            var service = scope.ServiceProvider.GetService<IScopedService>();
            _logger.LogInformation("Hello world!");
        }

        return Task.CompletedTask;
    }
}

**Comment by lahma** [comment]
Maybe save yourself some trouble and just use the [built-in support](https://www.quartz-scheduler.net/documentation/quartz-3.x/packages/hosted-services-integration.html)?

It may be that you have scheduled your job to run every second.

**Comment by jsonp80** [comment]
cronexpression: * 0/5 * * * ? 
also, it does wait 5 minutes before it make the first call "Execute" ,but once it does, it keep calling instead call once every 5 minutes.

**Reply by lahma** [reply]
You should enable logging and see why quartz takes this action. Is job throwing an exception and causing retry?
